### PR TITLE
Added board definition for Capable Robot Programmable USB Hub

### PR DESCRIPTION
**Pull request by osterwood**
This is the CircuitPython board definition for the upcoming Programmable USB Hub from Capable Robot Components.

More information about the product:
https://www.crowdsupply.com/capable-robot-components/programmable-usb-hub

CircuitPython firmware is being developed here: https://github.com/CapableRobot/CapableRobot_CircuitPython_USBHub_Firmware

### CODE REVIEW

**Comment by ladyada** [comment]
awesome :)

File: ports/atmel-samd/boards/capablerobot_usbhub/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,18 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash.ld
+USB_VID = 0x04D8
+USB_PID = 0xEDB3
```
Comment (tannewt) [inline ports/atmel-samd/boards/capablerobot_usbhub/mpconfigboard.mk]: Are you sure it's ok to use a MicroChip VID?

Comment (osterwood) [reply ports/atmel-samd/boards/capablerobot_usbhub/mpconfigboard.mk]: I went through the sublicense process and was approved.  

Screen Shot 2019-05-14 at 2 56 55 PM

Comment (tannewt) [reply ports/atmel-samd/boards/capablerobot_usbhub/mpconfigboard.mk]: Interesting! Thank you for letting me know.

**Pull request merged**